### DISCUSSION TITLE
Should frameworks be curated, too?

### DISCUSSION THREAD

**Discussion by octavian-nita** [SAP/curated-resources-for-domain-driven-design#14] [Q&A]
I haven't noticed any sections or references to frameworks that explicitly support or promote a DDD approach when architecting applications. Is it deliberate?

If that's not on purpose, here's a Java one I've been using so far: [Axon Framework](https://www.axoniq.io/axoniq-products#tab1) and its view on the matter: [Domain-Driven Design](https://developer.axoniq.io/domain-driven-design/overview).

**Comment by lechnerc77** [comment]
Up to now we wanted to focus on the methodology *per se* and keep technology aside. However, Let's discuss this topic here to see what others think about such an addition to the repository
